src/app/components/MediaLoader/README.md

```diff
@@ -1,6 +1,12 @@
 ## Description
 
-This component loads a media player into our webpage. It uses the BBC Universal Media Player module (BUMP) to choose the correct Embedded Media Player (EMP) for the user's device. Using BUMP removes the concerns involved in serving different devices and web-browsers as BUMP automatically provides the best fitting EMP for the client. For example, BUMP can cater for differences between mobiles, laptops and desktops, but also factors such as whether the user is using network data (3G) or a WIFI.
+Simple English:
+The MediaLoader takes data from the BFF and converts it into a settings object that BUMP can understand.
+
+BUMP (BBC Universal Media Player module) is a module that embeds videos into a webpage. Its an alternative to the standard <video> interface in HTML5.
+
+Long English:
+The MediaLoader uses the BBC Universal Media Player module (BUMP) to choose the correct Embedded Media Player (EMP) for the user's device. Using BUMP removes the concerns involved in serving different devices and web-browsers as BUMP automatically provides the best fitting EMP for the client. For example, BUMP can cater for differences between mobiles, laptops and desktops, but also factors such as whether the user is using network data (3G) or a WIFI.
```

Comment:
I don't know how helpful it is to go into an explanation of what BUMP does in our docs? Might be more helpful to keep it brief (like the sentence you have under 'simple english'). Would it be better to just do it all in one or two sentences not split by short/long - i.e `The MediaLoader uses the BBC Universal Media Player module (BUMP) to choose the correct Embedded Media Player (EMP) for the user's device.  The MediaLoader takes data from the BFF and converts it into a settings object that BUMP can understand.` as this focuses purely on what our component is doing

Reply:
( please feel free to ignore this though, little nit picky !! )

Reply:
You could link to SMP documentation about BUMP on Confluence instead of going into detail?